packages/nextjs/src/server/authMiddleware.ts

```diff
@@ -46,13 +46,13 @@ const INFINITE_REDIRECTION_LOOP_COOKIE = '__clerk_redirection_loop';
  * The default ideal matcher that excludes the _next directory (internals) and all static files,
  * but it will match the root route (/) and any routes that start with /api or /trpc.
  */
-export const DEFAULT_CONFIG_MATCHER = ['/((?!.*\\..*|_next).*)', '/', '/(api|trpc)(.*)'];
+export const DEFAULT_CONFIG_MATCHER = ['/((?!.+[\\w]+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'];
 
 /**
  * Any routes matching this path will be ignored by the middleware.
  * This is the inverted version of DEFAULT_CONFIG_MATCHER.
  */
-export const DEFAULT_IGNORED_ROUTES = ['/((?!api|trpc))(_next|.+\\..+)(.*)'];
+export const DEFAULT_IGNORED_ROUTES = ['/((?!api|trpc))(_next.*|.+[\\w]+\\.[\\w]+$)'];
```

Comment:
❓ why did we change it from `.next` to `_next.*`?

Reply:
The pattern has changed slightly, we now want to check for:
* Any paths that start with `/_next` (denoted by `_next.*`)
* Any paths that end in `*.*`

Previously we didn't care where the `*.*` was, so we had a `(.*)` after the capture group.